app/handler/provider_complaint.py

```diff
@@ -76,6 +162,17 @@ def get_original_message(cls, message: Message) -> Optional[Message]:
                 return part
         return None
 
+    @classmethod
+    def get_original_addresses(
+        cls, message: Message
+    ) -> Optional[OriginalMessageInformation]:
+        """
+        Try to get the proper recipient from original x-simplelogin-envelope-to header we add on delivery.
+        If we can't find the header, use the first address in the original message from"""
+        original = cls.get_original_message(message)
+        rcpt_header = original["x-simplelogin-envelope-to"]
```

Comment:
We can use `headers.SL_ENVELOPE_TO` here instead of `x-simplelogin-envelope-to`

Reply:
changed